api.ts

```diff
@@ -1592,6 +1631,7 @@ export function makeSyncTable<
       ...definition,
       cacheTtlSecs: 0,
       execute,
+      executeUpdate: executeUpdate as any,
```

Comment:
I could use some help on removing this `any` cast.  The `SchemaDefT` and `SchemaT` types are fighting given input vs. output types on the new method; unclear to me what the right fix is.